src/main/java/Duke.java

```diff
@@ -1,10 +1,318 @@
+    private static final String COMMAND_UNMARK = "unmark";
+    private static final String COMMAND_DELETE = "delete";
+
+    private static final int REMOVE_FROM = 5;
+    private static final int REMOVE_TO = 3;
+    private static final int REMOVE_BY = 3;
+
+    public static void main(String[] args) throws IOException {
+        File F =  new File("src/main/java/duke.txt");
+        if(!F.exists()){
+            F = new File("duke.txt");
+        }
+        printWelcomeMessage();
+        boolean canPrintFeedback;
+        Scanner in = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<Task>();
+        ArrayList<String> inputCommands = new ArrayList<String>();
+        try {
+            canPrintFeedback = false;
+            loadFileContents(F, tasks,inputCommands,canPrintFeedback);
+        }catch(FileNotFoundException e){
+            System.out.println("File does not exist.");
+        }catch(IOException e){
+            System.out.println("Could not read file.");
+        }
+        while (true) {
+            String userInput = in.nextLine();
+            canPrintFeedback = true;
+            processUserInput(tasks,inputCommands, userInput,F,canPrintFeedback);
+        }
+
+    }
+    private static void writeToFile(String filePath, ArrayList<String>inputCommands) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        for(String s:inputCommands){
+            fw.write(s);
+            fw.write('\n');
+
+        }
+        fw.close();
+    }
+
+    private static void appendToFile(String filePath, String userInput) throws IOException{
+        FileWriter fw = new FileWriter(filePath,true);
+        fw.write(userInput);
+        fw.write('\n');
+        fw.close();
+    }
+
+    private static void loadFileContents(File F, ArrayList<Task>tasks, ArrayList<String>commandInput,
+                                         boolean canPrintFeedback) throws IOException {
+
+        Scanner s = new Scanner(F);
+        while(s.hasNext()){
+            processUserInput(tasks,commandInput,s.nextLine(),F,canPrintFeedback);
+        }
+    }
+    private static String[] splitCommandAndDescription(String userInput) {
+        String[] commandAndArgs = userInput.split(" ", 2);
+        return commandAndArgs.length == 2 ? commandAndArgs : new String[]{commandAndArgs[0], ""};
+    }
+
+    private static void processUserInput(ArrayList<Task> tasks, ArrayList<String> inputCommands,String userInput,File F,
+                                         boolean canPrintFeedback) {
+        inputCommands.add(userInput);
+        String[] commandAndDescription = splitCommandAndDescription(userInput);
+        String command = commandAndDescription[0];
+        String description = commandAndDescription[1];
+        try {
+            switch (command) {
+            case COMMAND_BYE:
+                printByeMessage();
+                break;
+            case COMMAND_EVENT: //append
```

Comment:
Try to remove unnecessary comments. More on this will be covered in week 7. See [Week 7 - Code Quality - Comments](https://nus-cs2113-ay2223s2.github.io/website/schedule/week7/topics.html#w7-5-code-quality-code-comments)